api/v1alpha1/isMonitoringCRDExists.go

```diff
@@ -0,0 +1,33 @@
+package v1alpha1
+
+import (
+	kube "k8s.io/client-go/kubernetes"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func isMonitoringCRDExists(manager ctrl.Manager) bool {
+	config := manager.GetConfig()
+	clientset, err := kube.NewForConfig(config)
+	if err != nil {
+		databaselog.Error(err, "unable to get clientset while checking monitoring CRD")
+		panic(err)
```

Comment:
The operator must not `panic` here. Imagine the situation: cluster running for ages, and then the guy accidentally enables monitoring in the middle of the update, then in `ValidateUpdate` this function is called and boom, the operator is down, everyone cries)